### DISCUSSION TITLE
Does the tool support go application instrumentation for secure compilation protection, such as Relro and Strip?

### DISCUSSION THREAD

**Discussion by toptotu** [keyval-dev/opentelemetry-go-instrumentation#55] [Q&A] [answered]
Maybe the go binary does not have symbol table information.Does the tool support go application instrumentation for secure compilation protection, such as Relro and Strip?

**Comment by edeNFed** [comment, marked as answer]
Hi @toptotu 
Unfortunately not.
We need symbols in the ELF file to detect the memory addresses of functions to which we attach eBPF programs.
